test/compilable/readmodify_structclass.d

```diff
@@ -1,9 +1,4 @@
-/*
-REQUIRED_ARGS:
-TEST_OUTPUT:
----
----
-*/
+// REQUIRED_ARGS:
```

Comment:
Is `REQUIRED_ARGS` necessary in this case? Can't we delete that one also?

Reply:
I thought it was only fail_compilation and runnable where it was ignored, and compilable tests use the default permutations of flags unless overridden.